bootstrap2/app/views/kaminari/_page.html.erb

```diff
@@ -1,9 +1,3 @@
-<% if page.current? %>
-  <li class='active'>
-    <%= content_tag :a, page, remote: remote, rel: (page.next? ? 'next' : (page.prev? ? 'prev' : nil)) %>
-  </li>
-<% else %>
-  <li>
-    <%= link_to page, url, remote: remote, rel: (page.next? ? 'next' : (page.prev? ? 'prev' : nil)) %>
-  </li>
-<% end %>
+<li class="<%= 'active' if page.current? %>">
+  <%= link_to page, url, remote: remote, rel: (page.next? ? 'next' : (page.prev? ? 'prev' : nil)) %>
```

Comment:
This would make the link aways clickable even when it should not, a better way to fix this is to only update the `content_tag :a, ...` part.

Reply:
Sure thing! Thanks for the feedback. Will make the change shortly